package.json

```diff
@@ -58,7 +62,7 @@
   "devDependencies": {
     "babel-eslint": "7.2.3",
     "babel-plugin-check-es2015-constants": "6.22.0",
-    "babel-plugin-external-helpers": "6.22.0",
+    "babel-plugin-external-helpers": "https://registry.npmjs.org/babel-plugin-external-helpers/-/babel-plugin-external-helpers-6.22.0.tgz",
```

Comment:
Sorry - I didn't.  Must have been changed automatically as I was installing the npm modules.